docs/source/using/managing-minishift.adoc

```diff
@@ -19,37 +19,6 @@ The following sections contain information about managing the Minishift VM.
 For details about using Minishift to manage your local OpenShift cluster,
 see the link:../using/interacting-with-openshift{outfilesuffix}[Interacting with OpenShift] section.
 
-[[minishift-start-intro]]
```

Comment:
-1 for moving this to quick start, the basic life cycle methods and runtimes options are closely related. I am not sure what we try to achieve here?